bricks/util/test.cc

```diff
@@ -806,7 +806,7 @@ namespace object_with_interface_test_namespace {
 struct NamespacedObjectWithInterface {
   int calls = 0;
   int total_calls() { return ++calls; }
-  int add(int a, int b) const { return a + b; }  // `static` won't work with the inner `std::result_of_t<>`. -- D.K.
+  int add(int a, int b) { return a + b; }
```

Comment:
`static` then?

Reply:
It works with statics too now! That's a nice side effect of changing `std::result_of_t` into `std::invoke_result_t`.